Tests/AppcuesKitTests/Experiences/ExperienceLoaderTests.swift

```diff
@@ -68,7 +68,7 @@ class ExperienceLoaderTests: XCTestCase {
         let completionExpectation = expectation(description: "Completion called")
 
         // Act
-        experienceLoader.load(experienceID: "123", published: false, trigger: .preview) { result in
+        experienceLoader.load(experienceID: "123", published: false, queryItems: [], trigger: .preview) { result in
```

Comment:
Maybe add a `queryItem` here and then explicitly check it was added properly up on L55?